Azure/sonic-utilities
# [201911]: add show bgp neigh/network support for multi asic

Signed-off-by: Arvindsrinivasan Lakshmi Narasimhan <arlakshm@microsoft.com>



#### What I did
This is port of the PR https://github.com/Azure/sonic-utilities/pull/1574 in 201911 branch

This PR is to add support for the commands show ip bgp neighbor and show ip bgp network
Add unit tests for these commands

How I did it
Add change to get the bgp neighbor and network information from each namespace

How to verify it
Check the command works on single and multi asic platforms

UT  results
============================= test session starts ==============================
platform linux2 -- Python 2.7.13, pytest-3.0.6, py-1.4.32, pluggy-0.4.0
rootdir: /sonic/src/sonic-utilities, inifile: pytest.ini
plugins: cov-2.4.0
collected 314 items

sonic-utilities-tests/acl_config_test.py ............
sonic-utilities-tests/acl_loader_test.py .....................
sonic-utilities-tests/aclshow_test.py ..........
sonic-utilities-tests/counterpoll_test.py ...
sonic-utilities-tests/crm_test.py ..................................................
sonic-utilities-tests/db_migrator_test.py ..............................
sonic-utilities-tests/drops_group_test.py .......
sonic-utilities-tests/feature_test.py ...............
sonic-utilities-tests/filter_fdb_entries_test.py ..........
sonic-utilities-tests/intfstat_test.py ........
sonic-utilities-tests/intfutil_test.py .........
sonic-utilities-tests/ip_show_routes_multi_asic_test.py ...............
sonic-utilities-tests/ip_show_routes_test.py ......
sonic-utilities-tests/multi_asic_intfutil_test.py ..........
sonic-utilities-tests/pfcstat_test.py .......
sonic-utilities-tests/pfcwd_test.py ............
sonic-utilities-tests/port2alias_test.py ....
sonic-utilities-tests/portstat_test.py ..................
sonic-utilities-tests/psu_test.py ...
sonic-utilities-tests/sfp_test.py .........
sonic-utilities-tests/show_bgp_neighbor_test.py ............
sonic-utilities-tests/show_bgp_network_test.py .................
sonic-utilities-tests/show_ip_bgp_commands_test.py ...........
sonic-utilities-tests/show_ip_int_test.py ..........
sonic-utilities-tests/sku_create_test.py .....

========================= 314 passed in 65.28 seconds ==========================
make[1]: Leaving directory '/sonic/src/sonic-utilities/deb_dist/sonic-utilities-1.2'
 fakeroot debian/rules binary

#### Previous command output (if the output of a command-line utility has changed)

#### New command output (if the output of a command-line utility has changed)
